src/builder.ts

```diff
@@ -229,6 +231,7 @@ export interface BuilderConfig {
    * Otherwise, uses `printSchema` from graphql-js
    */
   customPrintSchemaFn?: typeof printSchema
+  features?: NexusFeatures
```

Comment:
If we put jsodc on `NexusFeatures` will that jsdoc apply to the `features` property here too if it has none?

Reply:
Unfortunately it does not seem to.